spec/active_job/limiter_spec.rb

```diff
@@ -50,7 +50,7 @@ def perform; end
         .with(instance_of(LimitedJob), expiration_time).and_return(false)
       expect(ActiveJob::Limiter).to_not receive(:clear_lock_before_perform)
       job = LimitedJob.perform_later
-      expect(job.job_id).to be_nil
+      expect(job == false || job.job_id == nil).to be true
```

Comment:
Tests were failing because of this. I guess perform_later stopped retuirning the job class?